### PR TITLE
refactor: receive screen

### PR DESCRIPTION
**Pull request by sandipndev**
The objective of this PR is to make the receive screen much better and in line with how things are dealt with in the send flow.

#### Todos
- [x] Delete individual receive usd and bitcoin screens
- [x] Refactor Payment Request and usePaymentRequest hook to be more in line with send
- [ ] Add three new pills - Invoice, On-chain and Paycode
- [ ] Let users pick between Bitcoin and USD (preselect default wallet) for LN Invoice
- [ ] Amountless or Invoice with Amount
- [ ] Add Note
- [ ] (when screenshot in LN Invoice) Display text: This invoice is only one time use and expires in 24 hours.

### CODE REVIEW

**Comment by nicolasburtey** [comment]
> * (when screenshot in LN Invoice) Display text: This invoice is only one time use and expires in 24 hours.

USD ln invoice expires after a few minutes

**Comment by UncleSamtoshi** [comment]
> Refactor Payment Request and usePaymentRequest hook to be more in line with send

Curious what you were thinking of changing with this as it was recently refactored. Would love to talk about the design of it before you make any changes.

File: app/screens/receive-bitcoin-screen/payment-requests/index.types.ts [hunk 1]
```diff
@@ -116,75 +116,8 @@ type BasePaymentRequest<T extends WalletCurrency> = {
...
-      unitOfAccountAmount: MoneyAmount<WalletOrDisplayCurrency>
-      settlementAmount: WalletAmount<T>
-    }
-  | {
-      unitOfAccountAmount?: undefined
-      settlementAmount?: undefined
-    }
-
-// TODO: Apply them
+// TODO: Apply Constraints to the type
```
Comment (nicolasburtey) [inline app/screens/receive-bitcoin-screen/payment-requests/index.types.ts:119]: why are you removing this code?

Comment (sandipndev) [reply app/screens/receive-bitcoin-screen/payment-requests/index.types.ts:119]: It's there in the next line. It still needs to have the constraints - it's underway

**Pull request closed without merging**